builder/cloudstack/config.go

```diff
@@ -34,6 +34,7 @@ type Config struct {
 	Hypervisor        string   `mapstructure:"hypervisor"`
 	InstanceName      string   `mapstructure:"instance_name"`
 	Keypair           string   `mapstructure:"keypair"`
+	TemporaryKeypair  string   `mapstructure:"temporary_keypair"`
```

Comment:
If I understand it correctly this field is not to supply a temporary keypair, but the name the Packer should use when creating a temporary keypair. I think this naming is a bit misleading.